### DISCUSSION TITLE
VScode Include InteliSense

### DISCUSSION THREAD

**Discussion by Gio2241** [xmake-io/xmake#4350] [Q&A]
I'm using xmake with cpp project:
```
add_requires("elfutils", "zlib")
add_requires("cxxopts", "nlohmann_json")
```

it compiles and works but VScode editor is unable to get includepaths by default.

how can fix it?
![Screenshot 2023-11-02 12:34:56](https://github.com/xmake-io/xmake/assets/132943010/08e00e3e-2dc6-4f75-a4c4-16b5905631b0)

**Comment by waruqi** [comment]
you need configure intellsense https://xmake.io/#/plugin/more_plugins?id=configure-intellsence

**Reply by Gio2241** [reply]
compile_commands is generated under .vscode but still same

**Reply by waruqi** [reply]
you need configure compile_commands to c_cpp_properties.json.

**Reply by Gio2241** [reply]
that was it
